include/alpaka/queue/Traits.hpp

```diff
@@ -78,5 +86,14 @@ namespace alpaka
                 ::empty(
                     queue);
         }
+
+        //-----------------------------------------------------------------------------
+        //! Queue based on the environment and a property
+        //
+        // Environment type can be a accelerator, device or a platform.
```

Comment:
Perhaps add `\tparam` comments, since this class is supposed to be used in a client app (and the updated examples do so)?
Also, `a accelerator` should be `an accelerator`.

Reply:
fixed